perf/istio-install/setup_istio.sh

```diff
@@ -170,9 +170,9 @@ function install_istio() {
       tar -xzf "${outfile}" -C "${DN}" --strip-components 1
       if [[ -z "${INSTALL_WITH_ISTIOCTL}" ]]; then
         mv "${DN}/install/kubernetes/helm" "${DIRNAME}/${release}"
-        mv "${DN}/bin/istioctl" "${DIRNAME}/${release}"
```

Comment:
@carolynhu you may need to keep this line because it is used for manual inject sidecar somewhere

Reply:
okay, I will recover it back in my next PR. However, the place to put it is not much reasonable. Could you please fix it when you get a time. Thanks